Python/Scrap_Latest_Torrents/README.md

```diff
@@ -0,0 +1,24 @@
+## Scrap Latest Torrents
+
+This script downloads the torrents in our Torrent client from the website https://rarbg.to/ from an entered label.
+
+### Requirements
+
+1.  You must have a torrent client, for example: uTorrent, BitTorrent, etc
+
+### Setup
+
+1.  Create a Virtual Environment.
```

Comment:
Can you brief about commands to create a virtual env in python? Would help beginners who are using this script.

Reply:
I add instructions